content/60.methods.md

```diff
@@ -2,179 +2,264 @@
+Additionally, new contributors have continued to join even as the project begins its second year.
+
+(Dot plot of contributions will go here) <!--To Do: Just adding a comment to add the caption and reference to figure here and also about in "Figure with dots". Also, I'm only counting 41 authors here, whereas 49 are mentioned earlier and later. Is that just because some folks contributed but did not use GitHub? If so, should note that.-->
+
+In order to make the project accessible to individuals from a range of backgrounds, we developed resources explaining how to use GitHub's web interface to develop and edit text and interact with Manubot for individuals with no prior experience working with git or other version control platforms.
+We developed tutorials containing visuals to explain how to open an issue, open a pull request, and review a pull request [@url:https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md; @url:https://github.com/greenelab/covid19-review/blob/master/INSTRUCTIONS.md].
+Additionally, the framework for evaluating literature was converted into issue templates to simplify the review of new articles.
+Articles were classified as _diagnostic_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-diag-study-template.md], _therapeutic_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-therapy-study-template.md], or _other_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-paper-template.md], with an associated template developed to guide the review of papers and preprints in each category.<!-- To Do: Number of paper issue and number of unique contributors?-->
+
+All seven manuscripts associated with this project will be submitted to _mSystems_ as part of a special issue that is providing support for evolving reviews, so that they can continue to be updated as more information becomes available.
+Currently, three manuscripts have been submitted, with one accepted for publication [@individual-nutraceuticals] and two under revision [@individual-pathogenesis; @individual-pharmaceuticals].
+The remaining four manuscripts are in preparation.
+These manuscripts cover a wide range of topics including the fundamental biology of SARS-CoV-2 (pathogenesis [@individual-pathogenesis] and evolution [@individual-evolution]), biomedical advances in responding to the virus and COVID-19 (pharmaceutical therapeutics [@individual-pharmaceuticals], nutraceutical therapeutics [@individual-nutraceuticals], vaccines [@individual-vaccines], and diagnostic technologies [@individual-diagnostics]), and biological and social factors influencing disease transmission and outcomes [@individual-inequality].
+To date, 49 authors are associated with the consortium (Figure @fig:projectstats).<!--To Do: break down by undergrad, grad/med student, post-doc, jr faculty?-->
```

Comment:
Is this still true? At least Vince got added to this paper for scite so we should make sure he rolls into the consortium.

Reply:
Daniel did too so we are either at 50 or 51, I have to check whether the Vince authorship merge was one of the commits used to generate this stat!

Reply:
OK Vince is 49, Daniel is 50, but I haven't merged in one other author so it's actually 51.
```suggestion
To date, 51 authors are associated with the consortium (Figure @fig:projectstats).<!--To Do: break down by undergrad, grad/med student, post-doc, jr faculty?-->
```